plotting/ft_interactivesourceviewer.m

```diff
@@ -0,0 +1,343 @@
+classdef ft_interactivesourceviewer<handle
```

Comment:
I don't see a better place for this file, but do propose to change its name. For high-level functions we tend to use ft_functionname, whereas low-level functions usually have multiple underscores. In this case I propose to start with ft_plot_xxx

Furthermore, I wonder whether it is "source" specific. That is a high-level concept and can be volumetric or surface-based. I suspect that this is more an interactive "mesh" plotter. A "mesh" is lower level and also used for other data (e.g. ft_plot_topo3d, ft_plot_mesh, or ft_meshrealign). Conceptually I think that a "source" can be a "mesh", but not the other way around; however, it is not always clear (considerer "raw+comp", "timelock+comp" and "freq+comp"). In this case I think the class expects a mesh with data, where data can have multiple dimensions.

How about calling it ft_plot_meshinteractive?

Reply:
Agreed, I'll call it `ft_plot_mesh_interactive`. Indeed the higher-level function is requesting 'source+mesh' data from ft_checkdata.